oecophylla/taxonomy/oecophylla-kraken.yaml

```diff
@@ -2,4 +2,4 @@ channels:
   - bioconda
 dependencies:
   - kraken
-  # - bracken
+  - bracken
```

Comment:
Is the Bracken auto installation issue already resolved?

Reply:
Not necessarily on Travis, but it is on Barnacle and we are no longer testing the environment installs on Travis.

Reply:
Got it!